packages/addons/vite-plugin-devtools-json/index.ts

```diff
@@ -0,0 +1,34 @@
+import { defineAddon } from '@sveltejs/cli-core';
+import { array, functions, imports, object, exports } from '@sveltejs/cli-core/js';
+import { parseScript } from '@sveltejs/cli-core/parsers';
+
+export default defineAddon({
+	id: 'vite-plugin-devtools-json',
+	alias: 'devtools-json',
+	shortDescription: 'Generate DevTools project settings in the devserver',
```

Comment:
Most of the other add-ons have a much shorter description. Maybe something like:
```suggestion
	shortDescription: 'chormium devtools',
```